### ISSUE TITLE
Date picker shows current month as a button but not clickable which conveys incorrect info to screenreader users

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#13848]
### Environment Information

- **Package version(s)**: uifabric/date-time@7.10.0
- **Browser and OS versions**: Edge + Narrator, Windows 2004 (19041.329)

### Describe the issue:

On date picker, there is a header showing the current selected month, like "June 2020". This header is implemented by <button> but it is not clickable. When Narrator is on, it reads this as "June 2020 button". This conveys wrong info to users that this button is clickable.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/yanbxice/pen/BajwqWo?editable=true%3Dhttps%3A%2F%2Ffabricweb.z5.web.core.windows.net%2Foufr%2F7.121.4%2F

#### Actual behavior:

Narrator reads the header as "June 2020 button"

#### Expected behavior:

It should be reads out as "June 2020"

**Comment by lorejoh12** [comment]
This button isn't actually focusable, it can only be accessed using Narrator's "Go to next item", but if you access it that way it does indeed read the wrong information.

I think the fix here is to change the item type from button -> div if it's not actually focusable. I don't see an appropriate role for a button that does nothing, and I don't think "Disabled" is appropriate here. Will add a fix for it.
